### ISSUE TITLE
Digg in your spatial tables with SQL and let DBeaver do the viz stuff 

### ISSUE THREAD

**Issue by gassenmj** [fun-with-maps/talks#2]
**Duration:** The talk should not be longer than 25 Minutes

**TL;DR:** i will demo how DBeaver can visualise various spatial data types and geometries on maps 

**Description**
DBeaver is a GUI Client to query databases (mostly) via SQL.
The nice thing about it: It doesn't care about the underlying database. In my case that means that i don't have to care about wether i query an Oracle database or a PostgreSQL database table, although spatial data is treated fundamentally different (PostGIS vs. sdo_geometry data type). Unfortunately DBeaver decided to provide map support for Oracle only in the Enterprise Edition. At work we operate both kinds of databases -> Oracle and PostgreSQL, i will give a little background about what we do and why we need spacial data and then show practical examples how i use DBeaver in my day to day job and why that visualisation is freaking helpful during my adhoc analysis.

**Comment by gassenmj** [comment]
i couldn't add the proposal label...

**Comment by boredland** [comment]
I would come around for that ;-)

**Comment by fabrik42** [comment]
This sounds good!
